File: src/main/java/com/dk/springboot/result/Result.java
package com.dk.springboot.result;

public class Result {
	private Long id;
	private Object data;
	private String message;
	private boolean status;
	private Long count;

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public Object getData() {
		return data;
	}

	public void setData(Object data) {
		this.data = data;
	}

	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}

	public boolean isStatus() {
		return status;
	}

	public void setStatus(boolean status) {
		this.status = status;
	}

	public Long getCount() {
		return count;
	}

	public void setCount(Long count) {
		this.count = count;
	}
}

File: src/main/java/com/dk/springboot/result/Result.scala
package com.dk.springboot.result


class Result extends Serializable {
    var id:Long = _

    var data:Any = _

    var count:Long = _

    var message:String = _

    var status:Boolean = false

    override def toString: String = id + ":" + data + ":" + count + ":" + message + ":" + status



}
